lib/modules/manager/asdf/extract.spec.ts

```diff
@@ -46,7 +46,8 @@ describe('modules/manager/asdf/extract', () => {
 
     it('can handle multiple tools in one file', () => {
       const res = extractPackageFile(
-        `awscli 2.8.6
+        `argocd 2.5.4
```

Comment:
```suggestion
        codeBlock`argocd 2.5.4
```

Please add the helper for better formatting

Reply:
Sorry, just to understand, do you want me to add `codeBlock` before the backtick?

Reply:
Honestly it was like this even before I touched it. I'm fine adding it but I just want to be clear my change was only about adding argocd.